### PR TITLE
Some wordsmithing / review

### PR DESCRIPTION
**Pull request by arnom-ms**
Per previous discussions the review as pull request with some minor changes. (relates to https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/36)

### CODE REVIEW

File: introduction/principles.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Let's take a look at each of these principles in more detail.
...
 
-In being open, a project is set up in such a way that contributing is as frictionless as possible.
-Projects should be discoverable and well documented.
-Anyone within an organization should be able to find them and ramp up without an inordinate amount of direct guidance from host team members.
+The configuration of an open project enables frictionless contributing.
+Projects should be discoverable and well documented through README.md and CONTRIBUTING.md files within the root of the repo.
+Anyone within an organization should be able to find a desired project and ramp up without an inordinate amount of direct guidance from host team members.
 Host team contact information should be prevalent with as many channels as makes sense for the project.
-Host team intentions to InnerSource their project should be broadcast through relevant organization channels.
+Host team intentions to InnerSource their project can be broadcast through relevant organization channels to raise awareness.
```
Comment (lenucksi) [inline introduction/principles.md]: Do we want to make any recommendation on marketing things here?

Comment (arnom-ms) [reply introduction/principles.md]: Done. Pls. let me know what you think.

File: introduction/principles.md [hunk 2]
```diff
@@ -33,27 +36,27 @@ it is, the more difficult it becomes for anyone to contribute.
...
 
 ## Prioritized Mentorship
 
 _Mentorship_ from host team to guest team via trusted committers is a key aspect of InnerSource.
-Contributors on guest teams are upleveled so that they understand enough about the host team system to change it successfully.
-In the process of doing so, they come to better understand the host team's software system as a general consumer.
-They can use it more effectively and help other consumers to do so as well.
+Contributors on guest teams are upleveled so that they understand enough about the host team's project/repo to change it successfully.
+In the process of doing so, they come to better understand the host team's software system as a general consumer and ambassador for the project/software.
+This individual committer can then over time and with experience take on a broader role in the project as a trusted committer.
```
Comment (rrrutledge) [inline introduction/principles.md]: Think it's important to use the InnerSource naming throughout.

```suggestion
This individual contributor can then over time and with experience take on a broader role in the project as a trusted committer.
```

Comment (arnom-ms) [reply introduction/principles.md]: You are soo correct. fixed now.

Comment (lenucksi) [reply introduction/principles.md]: Yup. If anyone has any additional suggestions please file them in a new PR.

**Comment by lenucksi** [comment]
> Folks good to merge this, now?

I guess @gruetter wanted to give this a review last week. 
Could you do this such that we can merge this on this weeks call?

**Comment by rrrutledge** [comment]
Let's merge by the time of the next call either way?

**Comment by lenucksi** [comment]
> Let's merge by the time of the next call either way?

Great idea, merging this then.
Should anyone want to add additional content or fixes, please make a new PR for this.

**Pull request merged**